reference/customers.v3.yml

```diff
@@ -2672,6 +2818,193 @@ components:
+            value:
+              type: string
+              description: The description for the metafield.
+            namespace:
+              type: string
+              description: |
+                Namespace for the metafield, for organizational purposes.
+              example: Sales Department
+              minLength: 1
+              maxLength: 64
+            permission_set:
+              type: string
+              description: |
+                Determines the visibility and writeability of the field by other API consumers.
+                | Value | Description |
+                | :--- | :--- |
+                | `app_only` | Private to the app that owns the field. |
+                | `read` | Visible to other API consumers. |
+                | `write` | Open for reading and writing by other API consumers. |
+                | `read_and_sf_access` | Visible to other API consumers, including on storefront. |
+                | `write_and_sf_access` | Open for reading and writing by other API consumers, including on storefront. |
+              enum:
+                - app_only
+                - read
+                - write
+                - read_and_sf_access
+                - write_and_sf_access
+            resource_type:
+              type: string
+              description: |
+                The type of resource with which the metafield is associated.
+              enum:
+                - brand
+                - product
+                - variant
+                - category
+                - cart
+                - channel
+                - location
+                - order
+                - customer
+              example: cart 
+            resource_id:
+              type: integer
+              description: |
+                The unique identifier for the resource with which the metafield is associated.
+              example: 424242
+              readOnly: true
+            description:
+              type: string
+              description: |
+                Description for the metafields.
+            date_created:
+              type: string
+              format: date-time
+              description: Date and time of the metafieldʼs creation.
+              example: '2022-06-16T18:39:00+00:00'
+            date_modified:
+              type: string
+              format: date-time
+              description: Date and time when the metafield was last updated.
+              example: '2022-06-16T18:39:00+00:00'
+            owner_client_id:
+              type: string
+              description: Client ID for the metafield's creator.
+              readOnly: true     
+          required:
+            - namespace
+            - key
+            - value
+            - permission_set
+            - resource_type
+            - resource_id
+            - description
+            - id
+            - date_created
+            - date_modified     
+    MetaFieldCollectionResponse:
```

Comment:
There seems to be another `MetafieldCollectionResponse` on [Line 4471](https://github.com/bigcommerce/docs/blob/5f54ab0b4d37da43eb707af515ed5c5dc10b77ba/reference/customers.v3.yml#L4471). If they are different, you might have to rename yours